tools/IoTEdgeDevOps/TestDashboard/README.md

```diff
@@ -0,0 +1,51 @@
+# High Level Overview
+Grafana is a tool where you can hook up data sources, query them, then visualize the queries via a GUI editor.
+
+The test dashboard is configured to pull from two data sources:
+Azure Monitor (Log Analytics): Used for longhaul / stress / connectivity test reports and metrics
+SQL Server: Used for imformation about builds / CI / E2E
+
+It then queries both of these to get the data and the visualization logic is built into the frontend.
+
+# Architecture
+There are two docker containers supporting the backend. The first is what is running the frontend grafana instance, and the second is a scheduled job ingesting data from Azure Dev Ops (one of the data sources) into a SQL server. A container to support the Azure Log Analytics data ingestion (the other data source) is not needed because grafana provides direct integration with this service.
+
+# Usage - Development
+
+This repository will allow you to spin up a local grafana dashboard for viewing and editing the test dashboard. In order to get started, first clone the repo, remove the .tmp extensions from all files, substitute the SQL server credentials in datasources.yaml, then run the following commands:
+```
+docker build -t azure-monitor-test .
+docker run -d -p 3000:3000 azure-monitor-test
+```
+
+Your grafana dashboard should now be accessible on localhost port 3000. The default username/password is admin/admin. Once signed in, you must add a datasource for Azure Monitor manually as grafana's integration here is currently broken.
```

Comment:
since you mentioned that grafana's integration is broken; please specify what version/build is using.

Reply:
Is this the same problem as I experienced or something else? does it make sense to add steps for `add a datasource for Azure Monitor manually`?

Reply:
I see... I guess you refer to the fact that you don't have Azure Monitor datasource in .json file and you had issues adding it. 

I still wonder why manually adding the Azure Monitor didn't work on my machine :(